src/ankit/arrayConcepts/SimpleCalculation.java

```diff
@@ -0,0 +1,70 @@
+package ankit;
+
+public class SimpleCalculation {
+	int index = 0;
+
+	int minimumNumber(int[] arr) {
+		int minimum = arr[0];
+		for (index = 0; index < arr.length; index++) {
+			if (minimum > arr[index])
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files